SerialPrograms/Source/CommonFramework/GlobalSettingsPanel.cpp

```diff
@@ -25,6 +25,7 @@ const std::set<std::string> TOKENS{
     "6643d9fe87b3e54dc75dfac8ac22f0cc8bd17f6a8a786debf5fc4c517ee65469",
     "8e48e38e49bffc8462ada9d2d9d850d5b3b5c9529d20978c09bc548bc9a614a4",
     "7694adee4419d62c6a923c4efc9e7b41def7b96bb84ea882701b0bf2e8c13bee",
+    "e8d168bc482e96553ea9f9ecaea5a817474dbccc2a6a228a6bde67f2b2aa2889",

```

Comment:
You can modify directly the code in SerialPrograms/Source/CommonFramework/GlobalSettingsPanel.cpp / .h to directly change DEVELOPER_MODE bool, much easier than adding your own token ;)